### PR TITLE
replacer: use errors.Wrap

### PR DESCRIPTION
**Pull request by keegancsmith**
This should help debug failures we encounter.

### CODE REVIEW

File: cmd/replacer/replace/replace.go [hunk 1]
```diff
@@ -222,24 +222,24 @@ func (s *Service) replace(ctx context.Context, p *protocol.Request, w http.Respo
...
 	stdout, err := cmd.StdoutPipe()
 	if err != nil {
 		log15.Info("Could not connect to command stdout: " + err.Error())
-		return
+		return false, errors.Wrap(err, "failed to connect to command stdout")
 	}
 
 	if err := cmd.Start(); err != nil {
 		log15.Info("Error starting command: " + err.Error())
-		return false, errors.New(err.Error())
```
Comment (keegancsmith) [inline cmd/replacer/replace/replace.go]: @rvantonder was it intentional to throw away the original error type?

Comment (rvantonder) [reply cmd/replacer/replace/replace.go]: Nope, but it is a side effect of me using [search.go](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@89a541042e0db909f6ee4320d6a51b462095a233/-/blob/cmd/searcher/search/search.go#L99) for inspiration.

**Comment by rvantonder** [comment]
Bunch of cases in [search.go](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@89a541042e0db909f6ee4320d6a51b462095a233/-/blob/cmd/searcher/search/search.go#L73-99) where we're not using `errors.Wrap` too

**Comment by keegancsmith** [comment]
I'll send a PR making us reflect the same changes in searcher.

**Pull request merged**